application/models/meeting.go

```diff
@@ -128,21 +128,21 @@ func (m *Meetings) FindOnDate(timeInFocus time.Time) error {
 }
 
 // FindOnOrAfterDate finds the meetings that have an EndDate on or after the timeInFocus-date
-func (m *Meetings) FindOnOrAfterDate(timeInFocus time.Time) error {
+func (m *Meetings) FindOnOrAfterDate(ctx context.Context, timeInFocus time.Time) error {
 	date := timeInFocus.Format(domain.DateTimeFormat)
 
-	if err := getOrdered(m, DB.Where("end_date >= ?", date)); err != nil {
+	if err := getOrdered(m, Tx(ctx).Where("end_date >= ?", date)); err != nil {
 		return fmt.Errorf("error finding meeting with end_date before %s ... %s", date, err.Error())
 	}
 
 	return nil
 }
 
 // FindAfterDate finds the meetings that have a StartDate after the timeInFocus-date
-func (m *Meetings) FindAfterDate(timeInFocus time.Time) error {
+func (m *Meetings) FindAfterDate(tx *pop.Connection, timeInFocus time.Time) error {
```

Comment:
Looks like this is only called from the tests. I see that it's different than FindOnOrAfterDate.

I wonder if they should have the same signature?

It looks like that is also true for FindOnDate.

Reply:
Changed, thanks.